### ISSUE TITLE
Could I re-train it with my own datasets?

### ISSUE THREAD

**Issue by haophancs** [amitshekhariitbhu/Android-TensorFlow-Lite-Example#7]

**Comment by ofirsteinherz** [comment]
Hi,
Yes you definitely can!
If the dataset is labeled, you should complete [tensorflow for poets](https://codelabs.developers.google.com/codelabs/tensorflow-for-poets/#0). Instead of download the training images from google, add yours.

Then, you have labels and NN (not tf-lite). So in [tensorflow for poets 2 tflite](https://codelabs.developers.google.com/codelabs/tensorflow-for-poets-2-tflite/#0) you will optimize your model using the TFLite converter. 

After that, just swich `mobilenet_quant_v1_224.tflite` and `labels.txt` with the files that tf for poets executed.

Good luck :-)

**Comment by bathanfritz** [comment]
I tried to retrain it with my own data set and used mobilenet_1.0_224 as tfhub module but it crashes, please help me

**Comment by soum-io** [comment]
My recent PR has added support for float models, which is probably the type of model you trained. Simply change the variable QUANT to false in TensorFlowImageClassifier.java along with changing the model and lebels name.

**Comment by sai-pher** [comment]
Hi there.
I wanted to know if you retrain with a new dataset, how do you order the new labels in the label.txt file? 
Does the order matter? And if so, how can one be sure of the order to use?

I working with a new model built using tesnorflow 1.13.1's Keras.
